app/components/MovieCard/MovieCard.css

```diff
@@ -1,20 +1,55 @@
 .movie-card {
-    border: 2px solid #8BD3E6;
-    border-radius: 15px;
-    box-shadow: 2px 2px 25px #767676;
-    text-align: center;
-    margin: 20px;
-    height: 250px;
-    width: 500px;
-    background-size: cover;
-    background-repeat: no-repeat;
-    opacity: .5;
+  border: 2px solid #8bd3e6;
+  border-radius: 15px;
+  box-shadow: 2px 2px 25px #767676;
+  text-align: center;
+  margin: 20px;
+  height: 250px;
+  width: 500px;
+  opacity: .5;
+  background-size: cover;
+  background-repeat: no-repeat;
+  animation: fadeIn ease-in 1;
+  animation-duration: .75s;
+  animation-fill-mode: forwards;
+  -moz-animation: fadeIn ease-in 1;
```

Comment:
Do we need all of this webkit stuff? Who uses firefox, anyway?

Reply:
I don't know, what do you think @jbexx ?

Reply:
I think my grandmother uses firefox.  @jackmallahan, do you not care about my grandmother?